source/winVersion.py

```diff
@@ -217,5 +217,9 @@ def isUwpOcrAvailable():
 	}
 
 
-def isFullScreenMagnificationAvailable():
+def isFullScreenMagnificationAvailable() -> bool:
+	"""
+	Technically this is always False. The Magnification API has been marked by MS as unsupported for
+	WOW64 applications such as NVDA. For our usages, support has been added since Windows 8.
+	"""

```

Comment:
I just realized that this method is in winVersion rather than screenCurtain, perhaps screen curtain specific comments aren't so appropriate. Otherwise I was going to suggest something like this:

```suggestion
	While the Magnification API has been marked by MS as unsupported for
	WOW64 applications such as NVDA. ScreenCurtain's specific usage of the API has been tested to confirm the approach works in released versions of Windows, this may not continue to be true in the future. The Magnification API was introduced by Microsoft with Windows 8.
	"""
```

Reply:
You could add some note to `ScreenCurtainProvider.canStart`